Telerik.Examples.ContentSecurityPolicy/Views/Shared/_Layout.cshtml

```diff
@@ -7,9 +7,10 @@
     <title>@ViewData["Title"] - Telerik.Examples.ContentSecurityPolicy</title>
 
     @{
-        var kendoVersion = "2023.3.1010";
+        var kendoVersion = "2025.2.520";
+        var themeVersion = "11.0.2";
     }
-    <link rel="stylesheet" href="https://kendo.cdn.telerik.com/themes/7.0.1/default/default-ocean-blue.css">
+    <link href="https://kendo.cdn.telerik.com/themes/@themeVersion/default/default-ocean-blue.css" rel="stylesheet" type="text/css" />
```

Comment:
Update the reference to the JSZip library (<script src="https://unpkg.com/jszip/dist/jszip.min.js"></script>)